fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -61,7 +61,7 @@ object FaciaContent {
 
     CuratedContent(
       content,
-      trailMetaData.headline.orElse(contentFields.get("headline")),
```

Comment:
I've checked this, and I think it's slightly different.

The Oracle database fields used:

`webTitle` --> `page_live.link_text`
`fields.headline` --> `content_live.headline`

(as seen here: https://github.com/guardian/api-indexer/blob/master/src/main/resources/sql/content-query-template.sql)

Most of the time they are the same, [but sometimes not] (http://content.guardianapis.com/lifeandstyle/wordofmouth/2015/jan/06/-sp-cup-soup-instant-taste-test-batchelors?api-key=test&show-fields=headline) -- the webTitle is used for the `<title>` (presumably there's some SEO benefit), and the headline for, uh, the headline. Not sure about fronts -- it's currently on the network front with a completely different (much shorter) title. On the `lifeandstyle/wordofmouth` tag page it appears to use either the headline, of another field set to exactly the same value.